app/routes.js

```diff
@@ -17,4 +18,6 @@ exports.init = app => {
     [validateBody(signupSchema), validateToken, validateRole(ADMIN)],
     users.signupAdmin
   );
+
+  app.post('/weets', [validateToken], weets.createWeet);
```

Comment:
Validate please the data sent in the body

Reply:
the endpoint doesn't receive data in the body 😅